scripts/prepare_sector_network.py

```diff
@@ -592,6 +595,34 @@ def update_wind_solar_costs(
             )
 
 
+def update_costs_tyndp(
+    costs: pd.DataFrame, tyndp_techs: dict[str, str]
+) -> pd.DataFrame:
+    """
+    Update technology and cost assumptions for TYNDP technologies based on TYNDP to PyPSA-Eur default carrier mapping.
+
+    costs : pd.DataFrame
+        Cost assumptions DataFrame
+    tyndp_techs : dict[str, str]
+        Dictionary of TYNDP technology names mapped to carriers.
+
+    Returns
+    -------
+    costs : pd.DataFrame
+        Updated cost assumptions DataFrame
+    """
+
+    for tyndp_tech, pypsa_tech in tyndp_techs.items():
+        costs.loc[tyndp_tech] = costs.loc[pypsa_tech]
+
+    # Add CCS assumptions as intermediate copy since there is no direct mapping atm
+    costs.loc["gas-ccgt-ccs"] = costs.loc["CCGT"]
+    costs.loc["coal-ccs"] = costs.loc["coal"]
+    costs.loc["lignite-ccs"] = costs.loc["lignite"]
```

Comment:
costs should be higher for CCS, you can find some assumptions e.g. for ccgt [here](https://github.com/open-energy-transition/open-tyndp/blob/5c290b69e5ed2d35fe0f22f0ce4b53dd20977367/scripts/prepare_sector_network.py#L1179-L1185)